src/web/app/pipes/is-identifier-anonymous.pipe.ts

```diff
@@ -0,0 +1,18 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { ANONYMOUS_PREFIX } from '../../types/feedback-response-details';
+
+/**
+ * check whether identifier is anonymous
+ */
+@Pipe({
+  name: 'isIdentifierAnonymous',
+})
+export class IsIdentifierAnonymousPipe implements PipeTransform {
```

Comment:
Pipe is used for transformation of a data to a displayed value. It is misused when it is used as a logical control.